### DISCUSSION TITLE
Existe-t'il une version graphique du schéma de la base de données ?

### DISCUSSION THREAD

**Discussion by AndrewHobson** [JeromeDevome/GRR#268] [Q&A]
Bonjour,
Je me demandais si il existe une version graphique de la représentation de la base de données Grr (version 3.4.3) qui montre toutes les tables, champs et les relations ?
Et si c'est le cas, on peut se la procurer où ?
En vous remerciant.
Andrew

**Comment by ynaessens** [comment]
Bonjour,
il y en a une dans la documentation (http://votreGRR/Doc) mais elle n'est pas à jour. L'essentiel y est cependant. Je l'avais fait à partir de phpmyadmin...
Cordialement,
YN

**Comment by AndrewHobson** [comment]
Merci.
C'est juste... A l'époque je l'avais trouvée.
C'est parfait, c'est ce qu'il me fallait.
